client.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2018 Twitch Interactive, Inc.  All Rights Reserved.
```

Comment:
maybe rename to `client_hooks.go`/`client_hooks_test.go` to be consistent with the server-side filenames?

Reply:
Ah, touché.

Reply:
I only ended up renaming `client.go` since `client_test.go` is in the `twirptest` package and it's a holistic test around the client, which hooks are a part of. Fixed in https://github.com/twitchtv/twirp/pull/198/commits/ee862b21bd2a8b1cc8eae533b8c1ba7fe74f19c4.